client/app/components/PopUp.js

```diff
@@ -25,13 +25,13 @@ class PopUp extends React.Component {
         NotificationManager.info(message);
         break;
       case 'success':
-        NotificationManager.success(message, 'Success', 1500);
+        NotificationManager.success(message, 'Success', 1300);
```

Comment:
thought experiment:

shouldn't the <NotificationContainer /> be always included (like in the main or board component), and thus the triggering of a specific notification (NotificationManager.success) be done within the redux action?

you would then get rid of the additional "smart" component "Popup"...